### ISSUE TITLE
how to prepare pre-trained model for custom dataset

### ISSUE THREAD

**Issue by Chuttyboy** [jfzhang95/pytorch-video-recognition#70]
Hai @jfzhang95 @sdxass 

I have tried the UCF dataset with X labels and its results are too good. Now how to detect an action for a custom dataset (own dataset)?

If I go with my own dataset how to prepare a pre-trained model? 

For example, I want  to detect shoplifting detection how to prepare a pre-trained model for this class 

Thanks
